specification/communication/data-plane/readme.md

```diff
@@ -84,7 +84,7 @@ These settings apply only when `--tag=package-2021-03-07` is specified on the co
 ```yaml $(tag) == 'package-2021-03-07'
 input-file:
   - Microsoft.CommunicationServicesIdentity/stable/2021-03-07/CommunicationIdentity.json
-  - Microsoft.CommunicationServicesCommon/stable/2021-03-07/common.json
+  - Microsoft.CommunicationServicesSms/stable/2021-03-07/communicationservicessms.json
```

Comment:
Bundling these swaggers together will cause the SDK code generation to include both of them in the same SDK.  Please confirm this is the desired behavior.

Reply:
@jhendrixMSFT how are we supposed to bundle them if we have multiple packages?

Context: We have a few packages:
* Identity
* Sms
* PhoneNumbers
* Chat

Reply:
A tag (`package-2021-03-07` here) defines a logical set of inputs.  When you build that tag, all inputs will be merged together, forming a single output (the analogy would be multiple .cpp files being compiled into the same binary).

The current authoring will combine `CommunicationIdentity.json` and `comminicationservicessms.json` into one document (conceptually speaking), so all of their APIs will be part of the same generated code.  If this is the desired behavior then leave it as-is.

Reply:
Also keep in mind that this composite authoring means the codegen _cannot_ independently version.

Reply:
@RezaJooyandeh , can you please confirm , this expected behavior , so that @jhendrixMSFT can merge this.

Reply:
I think we can go ahead with merging as the code generator currently is configured to point to the json file directly. We should do some more research to understand the tagging to change our sdk code to use the tags, and then we can update this file accordingly.

Reply:
Can you please elaborate on how the code generator points to the JSON?  What languages, how is that managed etc....  Your config file should represent the intent of how you build and ship your components (i.e. it should be _the_ source of truth).  It sounds like the intent is to build each swagger individually, the authoring should be updated to reflect that (it's just too easy to miss this later on and build/ship the wrong thing, then you're stuck supporting it).

Reply:
Here is an example: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/communication/Azure.Communication.Identity/src/autorest.md

We have a tracking issue to change that which I am guessing means we would be updating the tags here: https://github.com/Azure/azure-sdk-for-net/issues/18733

Reply:
@johanste do you have comments on this.